pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -445,11 +445,18 @@ func (a *ofFlowAction) Note(notes string) FlowBuilder {
 	return a.builder
 }
 
-func (a *ofFlowAction) SendToController(reason uint8) FlowBuilder {
+// SendToController will send the packet to the OVS controller.
+// If pause option is true, the packet will be not only sent to the controller, but
```

Comment:
will be sent to the controller and meanwhile also paused...

Reply:
Done.